o8g/Extractor/setguids.xml

```diff
@@ -0,0 +1,12 @@
+﻿<xml>
+  <sets>
+      <set id="gen-con-2017" number="00" value="b9353360-2f08-4517-a7d0-960b085d6ba4" />
+      <set id="core" number="01" value="7a1d7bfd-db29-4812-8de1-01cf7ca73323" />
```

Comment:
Might just be something I'm used to, but in conquest and previous l5r it either default ordered by set release date then card number or was possible by sorting the set column. Is this possible with thrones? Or a way to prefix the set name with its number?